google/cloud/aiplatform/datasets/dataset.py

```diff
@@ -42,6 +41,8 @@ class Dataset(base.AiPlatformResourceNounWithFutureManager):
     _resource_noun = "datasets"
     _getter_method = "get_dataset"
 
+    _support_metadata_schema_uris = None
```

Comment:
Prefer `_supported_metadata_schema_uris`. Also it would be nice to include the type signature to signal to devs what the supported type is, ie: `_supported_metadata_schema_uris: Optional[Tuple[str]] =  None`.